source/common/network/address_impl.cc

```diff
@@ -281,7 +288,7 @@ PipeInstance::PipeInstance(const std::string& pipe_path, mode_t mode,
     }
     pipe_.abstract_namespace_ = true;
     pipe_.address_length_ = pipe_path.size();
-    memcpy(&pipe_.address_.sun_path[0], pipe_path.data(), pipe_path.size());
+    memcpy(&pipe_.address_.sun_path[0], pipe_path.data(), pipe_path.size()); // NOLINT(safe-memcpy)
```

Comment:
Can you add a comment that this is safe because of the pipe-path size check above?

Reply:
Sure, I saw the if statement so I thought it was clear. Anyway, I will add a comment above this line.